test_utilities/src/common.rs

```diff
@@ -1510,3 +1544,34 @@ pub fn cluster_test(
 
     test_name.to_string()
 }
+
+pub fn metrics_server_test<P>(kubernetes_config: P, envs: Vec<(&str, &str)>) -> Result<(), SimpleError>
+where
+    P: AsRef<Path>,
+{
+    let result = kubernetes_get_all_hpas(kubernetes_config, envs);
+
+    match result {
+        Ok(hpas) => {
+            for hpa in hpas.items.expect("No hpa item").into_iter() {
+                if hpa.metadata.annotations.is_some() {
+                    if hpa
```

Comment:
can't you check something working? because if it's another error, we may have issues